crates/swc_css_ast/src/at_rule.rs

```diff
@@ -569,3 +571,201 @@ pub struct LayerNameList {
     pub span: Span,
     pub name_list: Vec<LayerName>,
 }
+
+#[ast_node("ContainerCondition")]
+#[derive(Eq, Hash, EqIgnoreSpan)]
+pub struct ContainerCondition {
+    pub span: Span,
+    pub name: Option<ContainerName>,
+    pub query: Vec<ContainerQuery>,
+}
+
+#[ast_node]
+#[derive(Eq, Hash, Is, EqIgnoreSpan)]
+pub enum ContainerName {
+    #[tag("CustomIdent")]
+    CustomIdent(CustomIdent),
+}
+
+#[ast_node]
+#[derive(Eq, Hash, Is, EqIgnoreSpan)]
+pub enum ContainerQuery {
+    #[tag("ContainerQueryNot")]
+    Not(ContainerQueryNot),
+
+    #[tag("ContainerQueryAnd")]
+    And(ContainerQueryAnd),
+
+    #[tag("ContainerQueryOr")]
+    Or(ContainerQueryOr),
+
+    #[tag("QueryInParens")]
+    QueryInParens(QueryInParens),
+}
+
+#[ast_node("ContainerQueryNot")]
+#[derive(Eq, Hash)]
+pub struct ContainerQueryNot {
+    pub span: Span,
+    pub keyword: Option<Ident>,
```

Comment:
I think we should call `eq_ignore_span` for this

Reply:
Oh is it sort of `raw`?

Reply:
We have:
```
impl EqIgnoreSpan for ContainerQueryNot {
    fn eq_ignore_span(&self, other: &Self) -> bool {
        self.query.eq_ignore_span(&other.query)
    }
}
```

because `keyword` is `raw`